src/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import logo from "../../images/buttonlogo.png";
+import "./HeroSection.css";
+
+const HeroSection = ({ img, title, text }) => {
+  const background = {
+    backgroundImage: `url(${img})`,
+  };
+
+  return (
+    <Container className="container" style={background}>
```

Comment:
this component does not handle showing the title,text and the button on the right